_scripts/evaluate_doc_retrieval.py

```diff
@@ -142,14 +151,15 @@ def _get_retrieved_doc_match(retrieved_doc_expected_col: str) -> pd.Series:
         # or
         # - equal to any of the actual retrieved documents (case sensitive)
         return (
-            (output_df[retrieved_doc_expected_col].isna())
-            |
-            (output_df[retrieved_doc_expected_col] == output_df["retrieved_doc1_actual"])
-            |
-            (output_df[retrieved_doc_expected_col] == output_df["retrieved_doc2_actual"])
-            |
-            (output_df[retrieved_doc_expected_col] == output_df["retrieved_doc3_actual"])
+                (output_df[retrieved_doc_expected_col].isna())
+                |
+                (output_df[retrieved_doc_expected_col] == output_df["retrieved_doc1_actual"])
+                |
+                (output_df[retrieved_doc_expected_col] == output_df["retrieved_doc2_actual"])
+                |
+                (output_df[retrieved_doc_expected_col] == output_df["retrieved_doc3_actual"])
         )
+
```

Comment:
I think all changes in this file are just formatting changes... can you confirm?

Reply:
yes, sorry about that